core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -41,8 +42,11 @@
                                        on-save-answer-group-feedback="saveActiveAnswerGroupFeedback"
                                        on-save-answer-group-dest="saveActiveAnswerGroupDest"
                                        on-save-answer-group-rules="saveActiveAnswerGroupRules"
+                                       on-save-answer-group-correct="saveActiveAnswerGroupCorrect"
                                        is-editable="editabilityService.isEditable()"
                                        display-feedback="!isLinearWithNoFeedback(answerGroup.outcome)"
+                                       labelled-as-correct="answerGroup.labelledAsCorrect"
+                                       default-answer-group=false
```

Comment:
Missing quotes around attr value (but see comments above, better to not include this in the first place).